crates/polars-core/src/serde/chunked_array.rs

```diff
@@ -174,3 +175,21 @@ impl Serialize for StructChunked {
         }
     }
 }
+
+impl Serialize for NullChunked {
+    fn serialize<S>(
+        &self,
+        serializer: S,
+    ) -> std::result::Result<<S as Serializer>::Ok, <S as Serializer>::Error>
+    where
+        S: Serializer,
+    {
+        {
+            let mut state = serializer.serialize_map(Some(3))?;
+            state.serialize_entry("name", self.name())?;
+            state.serialize_entry("datatype", self.dtype())?;
+            state.serialize_entry("values", &IterSer::new(std::iter::once(self.len())))?;
```

Comment:
It feels slightly hacky to encode the length under "values", but hey... it works!

Reply:
Luckily we don't serialize to human readable anymore. :wink: